src/NServiceBus.Transport.SQS/MessageDispatcher.cs

```diff
@@ -17,7 +18,7 @@ namespace NServiceBus.Transport.SQS
     using Settings;
     using Transport;
 
-    class MessageDispatcher : IMessageDispatcher
+    partial class MessageDispatcher : IMessageDispatcher
```

Comment:
Why do we need a partial here?

Reply:
The source gen for the regex requires that